assets/sass/components/_content-tile-small.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-hub-contentTileSmall {
```

Comment:
The rest of the tile CSS is in `_home-content.scss` rightly or wrongly. (A victim of evolution I think)
Should we try and keep this CSS together?

Reply:
i agree the tile css should be kept together. Whether that should be in the home-content scss is another matter. As the tiles are used in other areas i think a tile specific scss is a better approach. Happen to discuss this further.

Reply:
relocated class into home-content. I think this needs refactoring to break tile styles out into its own file.